.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: Test
+
+on: push
+
+jobs:
+  test:
+    name: Bootstrap app, run linters and tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [14, 16]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup node and restore cached dependencies
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+          cache: 'npm'
+
+      - name: Bootstrap React App
+        run: npx create-react-app test-app --template file:./
+
+      - name: Run test
```

Comment:
Should it be?
```suggestion
      - name: Run linters
```
```suggestion
      - name: Run test
```

Reply:
Fixed in [0cb178b](https://github.com/nimblehq/react-templates/pull/72/commits/0cb178b27c030f78ea36fbad2bd74ab1ee4a6811) along with the comments below 🙌🏼